### ISSUE TITLE
AuthorizationException from Terms of Service Acknowledgement

### ISSUE THREAD

**Issue by jonathanraftery** [jonathanraftery/bullhorn-rest-client#2]
Client will throw an AuthorizationException('Identity provider exception') if Terms of Service have not been acknowledged.

Typically this exception results from incorrect credentials, but currently is also thrown in the event above. Client needs a more meaningful exception in this case, or to automatically acknowledge the ToS.

###Current workaround
Manually perform OAuth login with the provided credentials, to clear the ToS acknowledgement.

**Comment by survivorbat** [comment]
Still having issues with this

**Comment by jonathanraftery** [comment]
Indeed, the issue is still open with no progress. Did the workaround work for you?

**Comment by survivorbat** [comment]
Yep, although it did take me some time to figure out that I had to be logged into BH in order for the website not to redirect me to the homepage.

**Comment by survivorbat** [comment]
#10

**Comment by survivorbat** [comment]
#10 Pull requuest made

**Comment by survivorbat** [comment]
Do you plan to intergrate it into the library? Or just keep it as a one-time inconvenience?
